.travis.yml

```diff
@@ -1,13 +1,10 @@
 sudo: false
 language: python
 python:
-  - '2.7'
```

Comment:
Remember to also remove `2.7` from the PyPI classifiers metadata in `setup.py`.

Reply:
Done.